components/sp-solutions/org.wso2.analytics.solutions.sp.message.tracer/widgets/OpenTracingVisTimeline/src/OpenTracingVisTimeline.jsx

```diff
@@ -242,84 +297,136 @@ class OpenTracingVisTimeline extends Widget {
-                            table = table +
-                                '<tr>' +
-                                '<td>No available baggage items</td>' +
-                                '</tr>'
-                        }
-
-                        return table;
+                        content = (
+                            <Card>
+                                <CardContent>
+                                    <Typography color="textSecondary" gutterBottom>Tags</Typography>
+                                    <Table>
+                                        <TableBody>
+                                            {
+                                                tagRows.map((row, index) => (
+                                                    <TableRow hover key={index}>
+                                                        <TableCell component="th" scope="row">
+                                                            <div>{row.key}</div>
+                                                        </TableCell>
+                                                        <TableCell>
+                                                            <div>{row.value}</div>
+                                                        </TableCell>
+                                                    </TableRow>
+                                                ))
+                                            }
+                                        </TableBody>
+                                    </Table>
+                                    {(tagRows.length <= 0) ? (<p style={{margin: 0}}>No Tags Found</p>) : (null)}
+                                    <br/>
+                                    <Typography color="textSecondary" gutterBottom>Baggage Items</Typography>
+                                    <Table>
+                                        <TableBody>
+                                            {
+                                                baggageItemRows.map((row, index) => (
+                                                    <TableRow hover key={index}>
+                                                        <TableCell component="th" scope="row">
+                                                            <div>{row.key}</div>
+                                                        </TableCell>
+                                                        <TableCell>
+                                                            <div>{row.value}</div>
+                                                        </TableCell>
+                                                    </TableRow>
+                                                ))
+                                            }
+                                        </TableBody>
+                                    </Table>
+                                    {(baggageItemRows.length <= 0) ? (<p style={{margin: 0}}>No Baggage Items Found</p>)
```

Comment:
Use `&&` instead of this as above suggestions.

Reply:
Fixed them in aa78bba